mu-plugins/blocks/sidebar-container/src/block.json

```diff
@@ -14,7 +14,7 @@
 		},
 		"inlineBreakpoint": {
 			"type": "string",
-			"default": "1200px"
+			"default": "1300px"
```

Comment:
Sidebar Container defaults are based on the ToC as that is the primary use case.